src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -118,8 +125,22 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
     }, {
         name: ColumnKey.STATUS,
         render: (trial: IDetailedTrialMatch) => (
-            <div className={styles.statusButton}>
-                <a target="_blank" href={"https://www.mskcc.org/cancer-care/clinical-trials/" + trial.protocolNo}><span className={styles.statusBackground}>{trial.status}</span></a>
+            <div className={styles.statusContainer}>
+                <a target="_blank" href={"https://www.mskcc.org/cancer-care/clinical-trials/" + trial.protocolNo}>
+                    <span className={styles.statusBackground}>{trial.status}</span>
+                </a>
+                <span className={styles.feedback}>
+                    <button className={"btn btn-default btn-sm btn-xs " + styles.feedbackButton} onClick={() => this.showTrialFeedback=true}>Feedback</button>
```

Comment:
You could use a Button component from react-bootstrap

Reply:
Changed to Button component